src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -355,9 +356,9 @@ const useGroupInfo = () => {
       Alert.alert(
         null,
         `Are you sure you want to remove ${
-          selectedUser.user.name || selectedUser?.user?.username
+          selectedUser.user.name || selectedUser?.user?.username || selectedUser.username
         } from this group? We will let the group know that you removed ${
-          selectedUser.user.name || selectedUser?.user?.username
+          selectedUser.user.name || selectedUser?.user?.username || selectedUser.username
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code for generating the alert message is repetitive and could be simplified by storing the username in a variable before using it in the template string.
```diff
- `Are you sure you want to remove ${
-   selectedUser.user.name || selectedUser?.user?.username || selectedUser.username
- } from this group? We will let the group know that you removed ${
-   selectedUser.user.name || selectedUser?.user?.username || selectedUser.username
- }.`,
+ const userName = selectedUser.user.name || selectedUser?.user?.username || selectedUser.username;
+ `Are you sure you want to remove ${userName} from this group? We will let the group know that you removed ${userName}.`,

```



<!-- This is an auto-generated comment by OSS CodeRabbit -->